build_tools/docker/bazel-tensorflow-nvidia/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # An image for building IREE with TensorFlow integrations using Bazel and
 # running Vulkan tests on an Nvidia GPU.
 
-FROM gcr.io/iree-oss/bazel-tensorflow-vulkan AS final
+FROM gcr.io/iree-oss/bazel-tensorflow-vulkan@sha256:caa0c9699f4041406bf978ed4b1ce69b3b40af436d6e999fd3fac037cd4d4749 AS final
```

Comment:
Huh not sure how I missed these....

Reply:
I added more to `cmake-*` in a PR last week too :P